libsplinter/src/network/connection_manager/mod.rs

```diff
@@ -216,6 +218,19 @@ impl NotificationHandler {
         }
     }
 
+    #[cfg(feature = "connection-manager-notifier-try-next")]
```

Comment:
The name of this feature doesn't match the function (try_listen) and try_listen doesn't make as much as much sense as calling it try_next.